src/pages/layer-2.tsx

```diff
@@ -40,153 +53,109 @@ import { getImage } from "../utils/image"
 import { GATSBY_FUNCTIONS_PATH } from "../constants"
+import { MdInfoOutline } from "react-icons/md"
+import { merge } from "lodash"
 
-// Styles
-
-const HeroBackground = styled.div`
-  width: 100%;
-  background: ${(props) => props.theme.colors.layer2Gradient};
-`
-
-const HeroContainer = styled.div`
-  width: 100%;
-`
-
-const Hero = styled(PageHero)`
-  padding-bottom: 2rem;
-`
-
-const PaddedContent = styled(Content)`
-  padding-top: 3rem;
-  padding-bottom: 3rem;
-`
-
-const LightGrayContent = styled(PaddedContent)`
-  background: ${(props) => props.theme.colors.layer2ContentSecondary};
-`
+type ChildOnlyType = {
+  children: ReactNode
+}
 
-const FlexContainer = styled.div<{ flexPercent: string | number }>`
-  flex: ${(props) => props.flexPercent}%;
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
-    flex: 100%;
+const SectionHeading = (props: HeadingProps) => {
+  const headingSpecificProps = {
+    fontSize: { base: "2xl", md: "2rem" },
+    fontWeight: 600,
+    lineHeight: 1.4,
   }
-`
 
-const Flex50 = styled.div`
-  flex: 50%;
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
-    flex: 100%;
+  if (props.as === "h3") {
+    headingSpecificProps.fontSize = { base: "xl", md: "2xl" }
   }
-`
 
-const StyledIcon = styled(Icon)`
-  fill: ${(props) => props.theme.colors.text};
-  margin-right: 0.5rem;
-  opacity: 0.8;
-  @media (max-width: ${({ theme }) => theme.breakpoints.l}) {
-  }
-  &:hover,
-  &:active,
-  &:focus {
-    fill: ${({ theme }) => theme.colors.primary};
-  }
-`
+  const mergeProps = merge(headingSpecificProps, props)
 
-const TwoColumnContent = styled.div`
-  display: flex;
-  justify-content: space-between;
-  gap: 4rem;
-  @media (max-width: ${({ theme }) => theme.breakpoints.l}) {
-    flex-direction: column;
-    align-items: flex-start;
-    margin-left: 0rem;
-    margin-right: 0rem;
-  }
-`
+  return <Heading {...mergeProps} />
+}
 
-const InfoGrid = styled(CardGrid)`
-  display: grid;
-  grid-template-columns: repeat(auto-fill, minmax(min(100%, 340px), 1fr));
-  gap: 1rem 2rem;
-  & > div {
-    height: fit-content;
-    &:hover {
-      transition: 0.1s;
-      transform: scale(1.01);
-      img {
-        transition: 0.1s;
-        transform: scale(1.1);
-      }
-    }
+const ContentBox = (
+  props: HTMLAttributes<"div"> & {
+    children: ReactNode
+    isLightGrayBg?: boolean
   }
-`
+) => (
+  <Box
+    px={8}
+    py={12}
+    width="full"
+    {...(props.isLightGrayBg && { background: "layer2ContentSecondary" })}
+  >
+    {props.children}
+  </Box>
+)
 
-const RollupCard = styled.div`
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-  background: ${(props) => props.theme.colors.ednBackground};
-  border-radius: 2px;
-  border: 1px solid ${(props) => props.theme.colors.lightBorder};
-  padding: 1.5rem;
-  flex: 50%;
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
-    flex: 100%;
-  }
-`
+const StyledInfoIcon = () => (
```

Comment:
Noting a difference in size on this icon

![image](https://user-images.githubusercontent.com/468158/220451363-8ce21725-6a57-414e-8b7e-36cee19e93b3.png)

vs prod version
![image](https://user-images.githubusercontent.com/468158/220451414-46344797-66ba-493d-9990-a7325082d78e.png)